clients/js/genjs.go

```diff
@@ -313,6 +322,11 @@ var methodTmplStr = `
       async.whilst(
         () => requestOptions.uri !== "",
         cbW => {
+	  this.logger.errorD("client-request-finished", {
+		"backend": "{{.ServiceName}}",
+		"message": arguments.length > 0 ? arguments[0] : "",
+		"status_code": 0
```

Comment:
that was just a test implementation. Actual implementation after talking with @cozmo https://github.com/Clever/wag/pull/160/commits/4c807adaa667887a308f23bae8f6b29bb017a416